spec/rubocop/cop/naming/ascii_identifiers_spec.rb

```diff
@@ -25,7 +23,7 @@
 
   it 'does not get confused by a byte order mark' do
     expect_no_offenses(<<-RUBY.strip_indent)
-      ﻿# encoding: utf-8
+      ﻿
       puts 'foo'
     RUBY
   end
```

Comment:
I think that `a byte order mark` is about `encoding magic comment`, so this test seems unnecessary already.

Reply:
The change was [introduced](https://github.com/rubocop-hq/rubocop/pull/354) a long time ago to protect against a [parser issue](https://github.com/whitequark/parser/issues/91).

Since the bug was fixed in parser and a regression spec added there, I guess it'd be fine to remove this spec, yeah.